src/ui/templates/insurance/your-business/company-details.njk

```diff
@@ -133,6 +133,13 @@
           },
           id: FIELDS.YOUR_COMPANY.WEBSITE,
           name: FIELDS.YOUR_COMPANY.WEBSITE,
+          value: submittedValues[FIELDS.YOUR_COMPANY.WEBSITE],
+          errorMessage: validationErrors.errorList[FIELDS.YOUR_COMPANY.WEBSITE] and {
+            text: validationErrors.errorList[FIELDS.YOUR_COMPANY.WEBSITE].text,
+            attributes: {
+              "data-cy": "company-website-error"
```

Comment:
Could be done in another PR, in this page can we use the field IDs where possible e.g

```js
"data-cy": FIELDS.POLICY_TYPE.ID + "-error-message"
```

Have this approach everywhere else

Reply:
yes please, another PR as there are a few to change